pubspec.yaml

```diff
@@ -7,13 +7,13 @@ publish_to: "none"
 
 environment:
   sdk: ">=2.18.5 <3.0.0"
-  flutter: 3.3.9
+  flutter: 3.10.0
 
 dependencies:
   ardrive_io:
     git:
       url: https://github.com/ar-io/ardrive_io.git
-      ref: v1.2.0
+      ref: PE-3934-modal-does-not-close-after-successful-or-failed-file-download-on-mobile
```

Comment:
Don't we have to update this? - I'm not sure if it's already released

Reply:
> Don't we have to update this? - I'm not sure if it's already released
you right! I'll update the PR